data/story northern union.txt

```diff
@@ -56,7 +56,7 @@ mission "Meeting an Alpha 0"
 	on complete
 		payment
 		dialog
-			`The warlord thanks you for the ride and pays you <payment>. As he is leaving, he says, "If you're interested in meeting my boss, head to a shop called "The Rusted Wheel" and find my commander. He's impossible to miss, try not to stare."`
+			`The warlord thanks you for the ride and pays you <payment>. As he is leaving, he says, "If you're interested in meeting my boss, head to a shop called "The Rusted Wheel" and find my commander. He's impossible to miss, though, try not to stare."`

```

Comment:
I think with a comma might also be correct, but I'd prefer without the comma.
Why? This way to me it reads as if "though" is part of the first part: "He's impossible to miss, though."
While it belongs to the last part: "though he's impossible to miss."